### PR TITLE
Made Grr approval urls visible in curses mode

### PR DESCRIPTION
**Pull request by ramo-j**
Currently GRR MPA urls are sent to the logger. This PR ensures that the url is also sent to `module.PublishMessage`, so in curses mode the operator still is able to retrieve it.

### CODE REVIEW

File: dftimewolf/lib/collectors/grr_base.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ def __init__(self) -> None:
     self.grr_url = str()
     self.approvers = []  # type: List[str]
     self.output_path = str()
+    self.message_callback: Callable[[str, bool], None] = None  # type: ignore
```
Comment (sydp) [inline dftimewolf/lib/collectors/grr_base.py:44]: Please add an attribute definition.

File: dftimewolf/lib/collectors/grr_base.py [hunk 2]
```diff
@@ -49,6 +50,7 @@ def GrrSetUp(
       grr_server_url: str,
       grr_username: str,
       grr_password: str,
+      message_callback: Callable[[str, bool], None],
```
Comment (sydp) [inline dftimewolf/lib/collectors/grr_base.py:53]: Similarly, add a docstring.

**Review by sydp — COMMENTED** [review]
Please update docstrings with new parameters but otherwise LGTM

File: dftimewolf/lib/collectors/grr_base.py (hunk 1 above)
Comment (ramo-j) [reply dftimewolf/lib/collectors/grr_base.py:46]: Done.

File: dftimewolf/lib/collectors/grr_base.py (hunk 2 above)
Comment (ramo-j) [reply dftimewolf/lib/collectors/grr_base.py:55]: Done.

**Review by sydp — APPROVED** [review]
LGTM

**Pull request merged**